swap/src/network/spot_price.rs

```diff
@@ -101,12 +101,12 @@ impl From<bitcoin::Network> for BitcoinNetwork {
     }
 }
 
-impl From<monero::Network> for MoneroNetwork {
+impl From<::monero::Network> for MoneroNetwork {
     fn from(network: monero::Network) -> Self {
         match network {
-            monero::Network::Mainnet => MoneroNetwork::Mainnet,
-            monero::Network::Stagenet => MoneroNetwork::Stagenet,
-            monero::Network::Testnet => MoneroNetwork::Testnet,
+            ::monero::Network::Mainnet => MoneroNetwork::Mainnet,
+            ::monero::Network::Stagenet => MoneroNetwork::Stagenet,
+            ::monero::Network::Testnet => MoneroNetwork::Testnet,
         }
     }
 }
```

Comment:
Do we still need this conversion and the enum `MoneroNetwork` now that we have a `serde` module that we can use to serialize an instance of `monero::Network`?

Reply:
This was inspired by what you said in a previous PR about thinking about the wire types. I am treating the wire network types separately so we don't accidently break the wire protocol because an underlying library changes. 
In case the `Network` struct of the Bitcoin and Monero library changes the wire protocol would change. If we assume that these enums never change I can use them directly - shall I change it?

Reply:
Good thinking!

The nice thing about using the new serde module is that it is still our code and will not change with updates to the library. Hence any potential breaking change will result in a compile error because the mapping no longer works.

Happy to have dedicated types as well if you find them easier. We are getting the same amount of safety out of a serde-module though :)